tests/system/action/motion/test_create.py

```diff
@@ -409,3 +409,1358 @@ def test_create_with_submitters(self) -> None:
         assert submitter_2.get("meeting_id") == 222
         assert submitter_2.get("user_id") == 57
         assert submitter_2.get("motion_id") == 1
+
+    def test_create_set_number_return_because_number_preset(self) -> None:
```

Comment:
Please put all numbering tests into an own file. There, you can create more structure specific for the numbering tests: Make some functions, that creates models. Make a small wrapper for `self.client.post`, so that not everything must always explicitly written. Maybe do multiple classes for different scenarios like manually, serially, per category. Choose better names for te tests (in a gooed named test class, you can just drop `issue24` and `motion_create`). And please merge your own tests into the ones described in the issue.

As far as I looked, I'm missing tests for `state.set_number=False` and maybe 1-2 tests for the `motion.set_state` action.

Reply:
Moved the mixin tests into an own file.

Reply:
Add a test for set_number False and motion. set_state.

Reply:
Not fully done yet. See the new comment below.